helm/regclient/values.yaml

```diff
@@ -94,8 +94,8 @@ startupProbe:
   httpGet:
     path: "/registration-client/{{ tpl .Values.regclient.version . }}/reg-client.zip"
     port: 80
-  initialDelaySeconds: 0
-  periodSeconds: 30
+  initialDelaySeconds: 420
```

Comment:
Instead of increasing initial delay let's increase the failure threshold.

Reply:
reverted initialdelay second and increased failure threshold @ckm007